syntax = "proto2";
import "game_event.proto";

message Event_DelCounter {
    extend GameEvent {
        optional Event_DelCounter ext = 2004;
    }
    optional sint32 counter_id = 1;
}
